config/service_config.go

```diff
@@ -245,3 +275,68 @@ func (c *ServiceConfig) getUrlMap() url.Values {
 
 	return urlMap
 }
+
+// GetExportedUrls will return the url in service config's exporter
+func (c *ServiceConfig) GetExportedUrls() []*common.URL {
+	if c.exported.Load() {
+		var urls []*common.URL
+		for _, exporter := range c.exporters {
+			url := exporter.GetInvoker().GetUrl()
+			urls = append(urls, &url)
+		}
+		return urls
+	}
+	return nil
+}
+
+// MockInitProviderWithSingleRegistry will init a mocked providerConfig
+func MockInitProviderWithSingleRegistry() {
```

Comment:
Could you move the mock data into test file or some mock data file?

Reply:
agree